src/components/OrganizationListGridItem.css

```diff
@@ -0,0 +1,39 @@
+:root {
+	--organization-list-grid-item--details-color: white;
+}
+
+.organization-list-grid-item {
+	padding: var(--fixed-spacing--1x);
+	color: inherit;
+	text-decoration: none;
+	border-bottom: 1px solid var(--color--gray--light);
+	line-height: 1.1;
+	display: flex;
+	flex-direction: column;
+	gap: var(--fixed-spacing--halfx);
+}
+
+.organization-list-grid-item:hover {
+	background-color: var(--color--white);
+}
+
+.organization-list-grid-item.active {
+	--organization-list-grid-item--details-color: var(--color--gray--darker);
+	background-color: var(--color--blue--lighter);
+}
+
+.organization-list-grid-item:last-child {
+	border-bottom: none;
+}
+
+.orginization-name {
+	grid-area: file-name;
+	font-weight: var(--font-weight--medium);
+	font-size: 1.1em;
+	word-wrap: break-word;
+}
```

Comment:
Looks like you snagged this styling from `BulkUploadListItem.css`. This should really mimic `.proposal--applicant-name` from `ProposalListGridItem.css`, which basically just means `font-weight`.